CMakeLists.txt

```diff
@@ -203,7 +202,10 @@ target_include_directories(libopensn
     $<BUILD_INTERFACE:${Boost_INCLUDE_DIRS}>
 )
 if (OPENSN_WITH_CUDA)
-    target_include_directories(libopensn PUBLIC $<BUILD_INTERFACE:${CUDA_INCLUDE_DIRS}>)
+    target_include_directories(libopensn
+        PUBLIC
+	$<BUILD_INTERFACE:${MPI_CXX_INCLUDE_DIRS}>
```

Comment:
Incorrect indentation

Reply:
I see this in the diff during review
<img width="518" alt="image" src="https://github.com/user-attachments/assets/1d6f766e-3bd9-4d1a-829b-81258cc44598" />
But looks ok on the PR page 🤔 
Can you check that there is no tab/space nonsense going on?

Reply:
Fixed.